### PR TITLE
faq: update backup and restore (#4362)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #4362 to release-4.0

---

### What is changed, added or deleted? (Required)

Update backup and restore.

Fixes pingcap/tidb#17787

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3425
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
@ireneontheway Please apply version-specific changes according to https://github.com/pingcap/docs/pull/3494.

File: faq/deploy-and-maintain-faq.md [hunk 1]
```diff
@@ -1,517 +1,515 @@
...
+
+#### 我们的 DBA 测试过 MySQL 性能，单台 TiDB 的性能没有 MySQL 性能那么好？
+
+TiDB 设计的目标就是针对 MySQL 单台容量限制而被迫做的分库分表的场景，或者需要强一致性和完整分布式事务的场景。它的优势是通过尽量下推到存储节点进行并行计算。对于小表（比如千万级以下），不适合 TiDB，因为数据量少，Region 有限，发挥不了并行的优势，最极端的就是计数器表，几行记录高频更新，这几行在 TiDB 里，会变成存储引擎上的几个 KV，然后只落在一个 Region 里，而这个 Region 只落在一个节点上。加上后台强一致性复制的开销，TiDB 引擎到 TiKV 引擎的开销，最后表现出来的就是没有单个 MySQL 好。
+
+### TiDB 备份恢复
+
+#### TiDB 主要备份方式？
+
+目前，数据量大时推荐使用 [BR](/br/backup-and-restore-tool.md) 进行备份。其他场景推荐使用 [Dumpling](/export-or-backup-using-dumpling.md) 进行备份。
```
Comment (ireneontheway) [inline faq/deploy-and-maintain-faq.md:482]:
```suggestion
目前，数据量大时推荐使用 [BR](/br/backup-and-restore-tool.md) 进行备份。其他场景推荐使用 [`dumpling`](/dumpling-overview.md) 进行备份。

```

File: faq/deploy-and-maintain-faq.md [hunk 2]
```diff
@@ -1,517 +1,516 @@
...
+#### 我们的 DBA 测试过 MySQL 性能，单台 TiDB 的性能没有 MySQL 性能那么好？
+
+TiDB 设计的目标就是针对 MySQL 单台容量限制而被迫做的分库分表的场景，或者需要强一致性和完整分布式事务的场景。它的优势是通过尽量下推到存储节点进行并行计算。对于小表（比如千万级以下），不适合 TiDB，因为数据量少，Region 有限，发挥不了并行的优势，最极端的就是计数器表，几行记录高频更新，这几行在 TiDB 里，会变成存储引擎上的几个 KV，然后只落在一个 Region 里，而这个 Region 只落在一个节点上。加上后台强一致性复制的开销，TiDB 引擎到 TiKV 引擎的开销，最后表现出来的就是没有单个 MySQL 好。
+
+### TiDB 备份恢复
+
+#### TiDB 主要备份方式？
+
+目前，数据量大时推荐使用 [BR](/br/backup-and-restore-tool.md) 进行备份。其他场景推荐使用 [`dumpling`](/dumpling-overview.md) 进行备份。
+
```
Comment (ireneontheway) [inline faq/deploy-and-maintain-faq.md:483]:
```suggestion

```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**